examples/src/main/scripts/submit-sql.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Usage: Run SQL statements
+#
+#    submit-sql.sh <spark_master_hostname>
+#
+
+SPARK_MASTER=${1:-localhost}
+
+curl -XPOST http://$SPARK_MASTER:6066/v1/submissions/create \
+--data '{
+  "appResource": "",
+  "sparkProperties": {
+    "spark.master": "local[2]",
+    "spark.driver.cores": "2",
+    "spark.driver.memory": "2g"
+  },
+  "clientSparkVersion": "",
+  "mainClass": "org.apache.spark.sql.hive.thriftserver.SparkSQLCLIDriver",
+  "action": "CreateSubmissionRequest",
+  "appArgs": [ "-e", "SHOW DATABASES; SELECT * FROM RANGE(10)" ]
```

Comment:
Hi @dongjoon-hyun

My first thought when I saw the filename was that I needed to pass in a sql, shall we parameterize this too?

Reply:
I got what you mean, @yaooqinn .

The `submit` comes from `submissions` of `http://$SPARK_MASTER:6066/v1/submissions/create`.

Since this is an example about how to submit simple statements like DDLs(CREATE TABLE...) or simple DMLs, I decided to keep this simple with static examples.

To be honest with you, `parameterized` script will be not reliable because the user-generated SQLs always requires special char escaping.

So, if you don't mind, let's keep this in AS-IS way. The users always get the idea and change it simply via text editors like VIM because this is REST API and JSON format.

Reply:
sgtm